src/effects/builtin/reverbeffect.cpp

```diff
@@ -64,7 +64,10 @@ EffectManifestPointer ReverbEffect::getManifest() {
     send->setName(QObject::tr("Send"));
     send->setShortName(QObject::tr("Send"));
     send->setDescription(QObject::tr(
-        "How much of the signal to send to the effect"));
+        "How much of the signal to send in to the effect\n"
+        "Lowering this fades out the effect smoothly\n"
+        "Use this to adjust the amount of the effect when the effect unit is in D/W mode\n"
+        "When the effect unit is in D+W mode, keep this turned up all the way"));
```

Comment:
The same here, the advice does not fit to the metaknob mapping.

Reply:
There is no mechanism to magically switch the metaknob linking with the mix mode, nor do I think there should be. I think the best way to improve the UX with the Send and Dry/Wet parameters of this effect will be to include default effect chain presets with Mixxx.

Reply:
Just remove the advise here. The Rest is OK. The meta knob mapping still works as desired even in D+W mode.